domain-models-runtime/src/test/java/org/folio/rest/persist/ddlgen/SchemaMakerTest.java

```diff
@@ -47,6 +47,29 @@ public void canCreateAuditedTable() throws IOException, TemplateException {
                              containsString("var1 TEXT;"), containsString("var2 TEXT;"), containsString("var3 TEXT;")));
   }
 
+  @Test
+  public void canCreateCompoundIndex() throws IOException, TemplateException {
+    SchemaMaker schemaMaker = new SchemaMaker("harvard", "circ", TenantOperation.UPDATE,
+      "mod-foo-18.2.3", "mod-foo-18.2.4");
+    String json = ResourceUtil.asString("templates/db_scripts/compoundIndexTest.json");
+    schemaMaker.setSchema(ObjectMapperTool.getMapper().readValue(json, Schema.class));
+    String result = schemaMaker.generateDDL();
+
+    assertThat(result, containsString("concat_space_sql(tablea.jsonb->>'firstName' , tablea.jsonb->>'lastName')"));
```

Comment:
Can you also check that schemaMaker.getSchema().getTables().get(0).get*Index().get(0).createCompoundIndex("tablea") is correct?

Reply:
The naming convention here is due to the choice of using the sql native function with a fixed  concat.  A new function is needed to guarantee the immutable value of the function for use in the index.